Hardware/11-Wifi.md

```diff
@@ -25,3 +25,155 @@ Top Layer
+
+![Wifi External Power](../assets/images/wifi_what_you_need.jpg)
+
+The Cyton is not able to supply enough current to power the power-hungry wifi shield, so we put a bigger voltage regulator on the shield to power both the Wifi chip and pass the Cyton components.
+
+## Interfacing with Ganglion
+
+![Battery Connection](../assets/images/wifi_ganglion_power.jpg)
+
+Install 4 AA batteries into your two battery packs or charge up your LiPo batteries. The Wifi shield has a 10x larger power draw then the Ganglion communication system so we recommend using LiPo to power the Wifi Shield! Please review _Powering The Shield_ below to properly power the system.
+
+### Seating the Wifi Shield
+
+Make sure the batteries are disconnected from both the Ganglion and the Wifi shield.
+
+To connect the Wifi shield to the Ganglion, first line up the big header pins on the Wifi shield to the Ganglion's female headers, like in the picture below:
+
+![Line up the big header](../assets/images/wifi_ganglion_seating_1.jpg)
+
+Then line up the two four pin headers.
+
+![Line up the other pins](../assets/images/wifi_ganglion_seating_2.jpg)
+
+Finally once all the pins are lined up, you can press straight down to fully seat the wifi shield onto it's new Ganglion home :)
+
+![Press down to seat](../assets/images/wifi_ganglion_seating_3.jpg)
+
+### Removing the Wifi Shield
+
+Remove power to your Ganglion and/or Wifi shield.
+
+![Wifi remove even fingers](../assets/images/wifi_ganglion_removing_1.jpg)
+
+Wiggle the board slowly off
+
+![Wifi wiggle](../assets/images/wifi_ganglion_removing_2.jpg)
+
+Or pull the shield straight off to avoid bending the pins.
+
+![Wifi off](../assets/images/wifi_removing_3.jpg)
+
+### Powering the Shield
+
+**IMPORTANT! Keep the dip switch labeled `EXT PWR` to `OFF` when using Ganglion.** As of today, the Wifi shield cannot pass through power to the Ganglion, therefore you must use two batteries.
+**Always turn the Wifi on before the Ganglion.** Your Ganglion cannot be powered from the Wifi shield, but the Ganglion can send reset signals to Wifi shield to reset it via software. Therefore you will need two sets of batteries (battery packs or LiPo) in order to use the Ganglion and Wifi shield together.
+
+![Wifi Power](../assets/images/wifi_battery_connection.jpg)
+
+When the Ganglion is on, the pass through LED on the Wifi shield will show the classic _pleasant_ blue LED when powered up!
+
+![Wifi Pass Through Power](../assets/images/wifi_ganglion_power_2.jpg)
+
+Verify that the external power switch is set to `OFF` to **NOT** power through to the Ganglion board.
+
+![Wifi External Power](../assets/images/wifi_ganglion_pass_through_power.jpg)
+
+The Ganglion is not able to supply enough current to power the power-hungry wifi shield, so we put a bigger voltage regulator on the shield. In the future, the wifi shield will be able to power the Ganglion too.
+
+## Sending Data to WiFi Shield
```

Comment:
I still don't know what this is. Is this getting data via wifi or is this similar to uploading code to the board?

Reply:
i moved this to a new document in openbci software called OpenBCI_Wifi_SDK to match the Ganglion and Cyton naming paradigm. It's for hooking up another micro-controller to the wifi shield.